packages/override-tools/src/Cli.ts

```diff
@@ -188,8 +188,6 @@ async function addOverride(overridePath: string) {
  * Remove an override from the manifest
  */
 async function removeOverride(overridePath: string) {
-  await checkFileExists('override', overridePath);
```

Comment:
>checkFileExists [](start = 8, length = 15)

Removed this so that we can delete overrides from the manifest easily if the actual file goes away